SPECS/python-twisted/python-twisted.spec

```diff
@@ -98,6 +101,9 @@ LANG=en_US.UTF-8 sudo -u test /home/test/.local/bin/tox -e nocov-posix-alldeps
 %{_bindir}/cftp3
 
 %changelog
+* Thu Aug 01 2024 Sindhu Karri <lakarri@microsoft.com> - 22.10.0-3
```

Comment:
The ptest seems to be failing: [link](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=615619&view=logs&j=bac295d9-f2e0-5165-e63e-e76f383e1a27&t=11f6dde0-e75e-52b2-fbf0-d83a00678fa3&l=2147)

Reply:
Hi @mfrw 
python-twisted ptests had been failing since the past 30+ builds. It is not a regression.
![image](https://github.com/user-attachments/assets/b7664c49-511c-4010-8781-de555e7d1eea)

Reply:
Related PR in 2.0: https://github.com/microsoft/azurelinux/pull/9989